src/rules/file/hasDoxygenHeader.spec.ts

```diff
@@ -86,7 +112,33 @@ describe('hasDoxygenHeader - fix', () => {
     expect(hasDoxygenHeader.fix!(content)).toEqual(content)
   })
 
-  it('should should add a doxygen header if not present', () => {
+  it('should update single asterisks to double if a doxygen header is already present', () => {
+    const contentOriginal = `
+    /*
+   @file
+   @brief Returns an unused libref
+ */
+
+ %macro mf_getuniquelibref(prefix=mclib,maxtries=1000);
+   %local x libref;
+   %let x={SAS002};
+   %do x=0 %to &maxtries;`
+
+    const contentExpected = `
+    /**
+   @file
+   @brief Returns an unused libref
+ */
```

Comment:
is it valid to end with a single asterisk?  Might need to test that in doxygen

Reply:
Yes, I have tested and confirmed from documentation: https://www.doxygen.nl/manual/docblocks.html#cppblock

Starting of doxygen header needs to be with double asterisks.

Reply:
seen - good shout